Module not found: Error: Can't resolve 'fast-deep-equal/react'

### Description

Suddenly, Google maps were giving this error on my production application 
<img width="621" alt="image" src="https://github.com/visgl/react-google-maps/assets/4696628/cc3f31df-caa4-44f1-b874-b07fa3bbedb7">

Therefore, I thought of checking for latest updates of the map's library and found out that my current version was 0.5.0 and the latest one released was 0.6.0. 
After upgrading the error in the console shown below is thrown and the app won't start.

### Steps to Reproduce

This happened after upgrading `@vis.gl/react-google-maps` from `0.5.0` to `0.6.0`

### Environment

- Library version: 0.6.0
- Google maps version: weekly
- Browser and Version: Chrome Version 121.0.6167.139 (Official Build) (arm64)
- OS: Apple M1 Max Ventura 13.2.1 (22D68)

### Logs

```text
ERROR in ./node_modules/@vis.gl/react-google-maps/dist/index.modern.mjs 3:0-48
Module not found: Error: Can't resolve 'fast-deep-equal/react' in '/Users/ari/Work/JS/REACT/dashboard-ui/node_modules/@vis.gl/react-google-maps/dist'
Did you mean 'react.js'?
BREAKING CHANGE: The request 'fast-deep-equal/react' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```

--- a/src/libraries/use-deep-compare-effect.tsx
+++ b/src/libraries/use-deep-compare-effect.tsx
@@ -1,5 +1,5 @@
 import {DependencyList, EffectCallback, useEffect, useRef} from 'react';
-import isDeepEqual from 'fast-deep-equal/react';
+import isDeepEqual from 'fast-deep-equal';
 
 export function useDeepCompareEffect(
   effect: EffectCallback,

